packages/ruby/spec/readme/metrics_spec.rb

```diff
@@ -15,7 +15,7 @@
 
   context "in a multi-threaded environment" do
     it "doesn't wait for the HTTP request to Readme to finish" do
-      readme_request_completion_time = 1 # seconds
+      readme_request_completion_time = 2 # seconds
```

Comment:
still seeing some flakiness, but I think this made things a little better as it seems to be failing less frequently from what I've seen so far 🤞